arches/app/etl_modules/base_data_editor.py

```diff
@@ -360,7 +362,8 @@ def run_load_task(self, loadid, graph_id, node_id, operation, language_code, old
                 return {"success": False, "data": {"title": _("Error"), "message": data_staged["message"]}}
 
         if data_updated["success"]:
-            data_updated = self.save_to_tiles(loadid, finalize_import=False)
+            self.loadid = loadid  # currently redundant, but be certain
```

Comment:
Still, it seemed wise to just add these belt-and-suspenders redundant statements just to be safe.